update.py

```diff
@@ -132,22 +133,42 @@ class HealthStatus:
     run_data: RunData = field(default_factory=RunData.get)
 
     async def run(self) -> None:
-        summary: dict[str, dict[str, str]] = {}
+        all_reports: list[DandisetReport] = []
         async for dandiset in self.aiterdandisets():
             log.info("Found Dandiset %s", dandiset.identifier)
             report = await dandiset.test_assets()
             await report.dump(self.reports_root / dandiset.identifier, self.run_data)
-            summary[dandiset.identifier] = report.summary()
+            all_reports.append(report)
         async with await (self.reports_root / "README.md").open("w") as fp:
```

Comment:
hard to assess how README would look like by just looking at the code... eventually might be worth separating out running the tests across dandisets from rendering the summary into two separate functions and have a dedicated invocation for re-rendering of README... but ok -- for now let's just merge!